### ISSUE TITLE
UTF8 vs byte mismatch in "insert-text" message

### ISSUE THREAD

**Issue by stacksmith** [crategus/cl-cffi-gtk#52]
I am not sure what to do with this one, but it wasted a lot of my time.
The Lisp handler is invoked after CFFI converts the string to Lisp.  But the len parameter is a byte count, the meaning of which is now lost, since we don't have bytes anymore.  I think a character count would be more correct...

**Comment by Ferada** [comment]
Agreed this is an edge case. Perhaps the string conversion shouldn't actually be done automatically in case the user doesn't want that to happen?

Btw. are you on freenode? I'm wondering if work on this library can be a bit more organised, so feel free to come chat (I'm using `ferada` on there), I don't want to misappropriate the issue tracker here too much.
